src/index.d.ts

```diff
@@ -1023,6 +1155,30 @@ declare namespace admin.remoteConfig {
      */
     publishTemplate(template: RemoteConfigTemplate, options?: { force: boolean }): Promise<RemoteConfigTemplate>;
 
+    /**
+    * Rolls back a project's published Remote Config template to the specified version.
+    * A rollback is equivalent to getting a previously published Remote Config
+    * template and re-publishing it using a force update.
+    * 
+    * @param versionNumber The version number of the Remote Config template to rollback to.
+    *    The specified version number must be lower than the current version number, and not have
+    *    been deleted due to staleness.
```

Comment:
What's the limit on this?  300 versions/90 days?

Might be good to mention here, not sure.  It's right below as well :)

Reply:
Added 300 versions/90 days text here as well